helpers/lib/fonts.js

```diff
@@ -84,58 +86,76 @@ const fontProviders = {
  * @returns {Object.<string, Array>|string}
  */
 module.exports = function(format, themeSettings, handlebars, options) {
-    
+
     const collectedFonts = {};
-    _.each(themeSettings, function(value, key) {
-        //check that -font is on end of string but not start of string
+    const themeSettingsKeys = Object.keys(themeSettings);
+    for (let i = 0; i < themeSettingsKeys.length; i++) {
+        const setting = themeSettingsKeys[i];
```

Comment:
I would put
```
const key = themeSettingsKeys[i];
const value = themeSettings[key];
```
at the top of the loop and that will minimize changes below and probably make things easier to read

Reply:
done.